### DISCUSSION TITLE
Welcome to deepnest Discussions!

### DISCUSSION THREAD

**Discussion by Dexus** [deepnest-next/deepnest#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tiktuk** [comment]
Hi Dexus,

Great that you've started this new work on DeepNest, it's very much appreciated!

I came here because I was wondering if it's possible to integrate DeepNest with Fusion 360 somehow. It would be great if one could (via a Fusion add-in) send an svg representation of the components to DeepNest and get the resulting transformations back, without exporting and importing manually. [FuseNest](https://github.com/phossystems/FuseNest)  does something like this, embedding SVGNest in a Fusion add-in web view, but I don't think this is possible with DeepNest, as it's a Node project.

So I was wondering (I'm not familiar with Electron apps): Would it be possible to communicate with DeepNest via IPC or a REST api?

PS: I could not figure out how to create a new thread any other place than her, feel free to move this to a more appropriate place.

**Reply by Dexus** [reply]
Hi tiktuk,

Thank you for that.

I think it would work in general. But not at the stage we are at right now. But I can put it on the agenda, if you like. But I guess it will be next year or 2026 if we have to do it without sponsorship.

Currently there is no REST API or IPC available. Except for the one offered by Electron, which is probably not accessible from the outside. At the moment there are thoughts about cloud services. These would be accessible via API. But I am not yet there.

Many greetings
josef Fröhle

**Reply by tiktuk** [reply]
Thanks for your quick reply, and for considering it. I would be happy if it was on the agenda, could also be useful for many other pieces of software :) . I will look more into it as well.

**Reply by Dexus** [reply]
You're welcome. Will add it to my roadmap.
